src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,631 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
```suggestion
// Some of operators is not compatible with naming convention.
// To prevent this, let's disable eslint for this file.
/* eslint-disable @typescript-eslint/naming-convention */

// TODO Add empty options as comment

```

Reply:
"Please add TODO comment" was "in the source code" :)